custom_components/tuya_local/devices/dowell_dchs-k7_32_7kw_ev_charger.yaml

```diff
@@ -0,0 +1,565 @@
+name: EV charger
+products:
+  - id: bf2dedfe80f14bc81czzwo
+    manufacturer: Dowell Technology Co., Ltd.
+    model: DCHS-K7    # (Dowell iOneAIO ESS - 32A - 7kW)
+entities:
+  - entity: sensor
+    translation_key: status
+    icon: "mdi:ev-station"
+    class: enum
+    dps:
+      - id: 3    # work_state
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: charger_charging
+            value: charging
+          - dps_val: charger_free
+            value: available
+          - dps_val: charger_insert
+            value: plugged_in
+          - dps_val: charger_free_fault
+            value: fault_unplugged
+          - dps_val: charger_wait
+            value: waiting
+          - dps_val: charger_pause
+            value: paused
+          - dps_val: charger_end
+            value: charged
+          - dps_val: charger_fault
+            value: fault
+  - entity: sensor
+    name: meter_id
+    category: diagnostic
+    dps:
+      - id: 22    # meter_id
+        type: string
+        name: sensor
+  - entity: sensor
+    category: diagnostic
+    name: system_version
+    dps:
+      - id: 23    # system_version
+        type: string
+        name: sensor
+  - entity: sensor
+    name: Total electricity consumption

```

Comment:
I'd suggest to remove the name and let the class default name shine through, with its translations.